### PR TITLE
New code signing & more silly fixes

### PR DESCRIPTION
**Pull request by Perksey**
- [x] #922 
- [x] #1264 
- [x] #1307 
- [x] #1350 
- [x] Implicit count not inverting calculation in count
    - [x] Add expression recognition
    - [x] Invert expression
- [x] Integrate new code signing tool
- [x] Use new code signing tool in CI/CD
- [x] Update patch notes

### CODE REVIEW

**Comment by Perksey** [comment]
Ready for review.

File: src/Core/Silk.NET.Core/Native/SilkMarshal.cs [hunk 1]
```diff
@@ -142,9 +142,8 @@ public static int GetMaxSizeOf(string? input, NativeStringEncoding encoding = Na
             => encoding switch
             {
                 NativeStringEncoding.BStr => -1,
-                NativeStringEncoding.LPStr => ((input?.Length ?? 0) + 1) * Marshal.SystemMaxDBCSCharSize + 1,
-                NativeStringEncoding.LPTStr => (input is null ? 0 : Encoding.UTF8.GetMaxByteCount(input.Length)) + 1,
-                NativeStringEncoding.LPUTF8Str => (input is null ? 0 : Encoding.UTF8.GetMaxByteCount(input.Length)) + 1,
+                NativeStringEncoding.LPStr or NativeStringEncoding.LPTStr or NativeStringEncoding.LPUTF8Str
+                    => (input is null ? 0 : Encoding.UTF8.GetMaxByteCount(input.Length)) + 1,
```
Comment (Beyley) [inline src/Core/Silk.NET.Core/Native/SilkMarshal.cs:146]: isnt LPTStr different if `UNICODE` is defined? what does silk prefer for windows functions?
(eg. LPTStr is a pointer to a 16bit number when `UNICODE` is defined but a pointer to an 8bit number if not)

**Review by Beyley — APPROVED** [review]
seems to lgtm aside from one small question

File: src/Core/Silk.NET.Core/Native/SilkMarshal.cs (hunk 1 above)
Comment (Perksey) [reply src/Core/Silk.NET.Core/Native/SilkMarshal.cs:146]: You're right however we've the LPStr/LPTStr/LPUTF8Str enums are just for compatibility and aren't really intended to be used (we're using them here out of habit :/). The Ansi/Auto/Uni/UTF8 ones are designed to be used.

Silk.NET has never observed this LPTStr behaviour and may be confusing to users if we changed this. Admittedly this was an oversight, but a 3 year old one that noone seems to have kicked up a fuss about

**Pull request merged**